### ISSUE TITLE
Nightly and debug builds should have different icons

### ISSUE THREAD

**Issue by julioromano** [vector-im/element-x-android#1089]
### Steps to reproduce

1. Install both a nightly build and a local development build.
2. Notice they have the same icons so it's difficult to understand which is which form the launcher.

#### What did you expect?
Nightly builds have a "nightly" mark in their launcher icon
Debug builds have a "dbg" mark in their launcher icon

### Will you send logs?

No

### Are you willing to provide a PR?

No

**Comment by julioromano** [comment]
@bmarty would it be possible to have also another background color for the "dev" builds? (i suspect with the recent change nightlies and dev builds are indistinguishable)

**Comment by bmarty** [comment]
New icons:

Prod / Debug / Nightly

<img width="227" alt="image" src="https://github.com/vector-im/element-x-android/assets/3940906/1e0e86c6-b2ea-4deb-b798-a812ac30dbb0">
